MineLeapMod/build.gradle

```diff
@@ -55,6 +52,10 @@ processResources {
 
 }
 
+runClient {
+    //systemProperty "java.library.path", "/usr/lib/Leap"

```

Comment:
I keep it for development purpose. It's easier to write the path here once rather than write it in the GUI at each start of Minecraft.